content/channels/index.textile

```diff
@@ -447,6 +447,72 @@ To publish a single message to multiple channels, make multiple "@publish()@":/a
 <p>A client can choose whether or not to receive messages that they publish themselves using the "@echoMessages@":/api/realtime-sdk/types#client-options property. This is set in @ClientOptions@ with a default value of @true@. Setting @echoMessages@ to @false@ will stop clients receiving the messages they publish themselves. This property is only available using the realtime interface, as it isn't possible to subscribe to messages using the REST interface.</p>
 </aside>
 
+h3(#server-side-batching). Server-side batching [beta]
+
+Server-side batching optimizes message processing and delivery by grouping multiple messages into batches before sending them to subscribers. This feature reduces the overall message count, cutting costs and mitigating the risk of hitting rate limits during high-throughput scenarios.
+
+h4. Key benefits
+
+* Cost efficiency.
+* Reduced system load.
+* Message rate limit mitigation.
+
+h4. Use cases
+
+The are the primary use cases where server-side batching provides significant value:
+
+* *High-volume chat applications:*
+A live event chat platform experiences large spikes in message volumes during peak moments, such as live sports games or concerts. Managing high-frequency chat messages and reactions from thousands of users simultaneously can increase costs and the risk of hitting message rate limits.
+_*Solution:*_ Implementing server-side batching allows the platform to group multiple chat messages and reactions into single batches before delivering them to subscribers. This reduces the number of individual messages processed and helps maintain a seamless and interactive user experience during high-activity periods.
```

Comment:
These are good, but I'm not sure if it works having a separate 'solution' called out, rather than trying to achieve the same thing with bullet points or less text?

Reply:
I have removed "solution" and reduced the word count